specs/file_share_list_view.adoc

```diff
@@ -0,0 +1,143 @@
+// vim: tw=79
+
+= Tendrl Definitions: Display list of file shares present in Gluster cluster in Tendrl
+
+Display list of file shares associated with a Gluster cluster present in Tendrl. The list should have functionality for 
+
+* filtering by name, cluster,status and type.
+* sorting by name and utilisation.
+
+File share list will be sorted by name by default.
+
+== Problem description
+
+Tendrl should show list of file share associated with a gluster cluster along with their details.
+
+The details shown for each file share in UI is -
+
+* Name
+* Status
+* 2-way/3-way distributed
+* Cluster Name
+* CPU Utilisation
+* Bricks
+* Active alerts
+* Last Rebalance date
+
+API support present for -
+
+* Name
+* Status
+* Cluster Name
+* CPU Utilisation
+* Bricks
+
+== Use Cases
+
+Users can see the list of file shares associated with a gluster cluster present in Tendrl. They can also 
+
+* filter by name, cluster, status and type.
+* sort by name and utilisation.
+
+As per API support, users can 
+
+* filter by name, cluster and status.
+* sort by name and utilisation.
+
+== Proposed change
+
+* Create a view for listing file share associated with a gluster cluster.
+* For utilisation, use donut chart component.
+* Implement client side filer and sorting functionality.
+* Filter call will trigger after 1 sec the user enters the filter data.
+* Send polling call after 10 minutes to get the updated data. 
+* Show total count of file shares present in a gluster cluster.
+
+=== Alternatives
+
+Generate list component can be created and used for listing the file shares. But because of its complexity and time constraint, we will pick this up later.
+
+=== Data model impact:
+
+
+=== Impacted Modules:
+
+==== Tendrl API impact:
```

Comment:
This part lack information. state what is available currently(cluster wide list API available), what extra information reqd in the current API. Also specify the any extra APIs required(syatem wide listing?)
Which component implements sorting and filtering?